### DISCUSSION TITLE
2023-03-15 Weekly Meeting 99 Agenda/Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#370] [General]
Weekly Meeting 99 is happening on Wednesday, March 14, 2023 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Review the action items from retro 

### Discussions: 
- new WG proposals 
- [OMI 100](https://github.com/omigroup/omigroup/discussions/371) is next week! 

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
  - shifting to 501c, applying for a grant (Epic Mega Grants), MSF-update itself 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - update draft protocol submission #356 
  - omi_personality
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research](https://github.com/omigroup/ux-research) 
- [OMI Website update](https://github.com/omigroup/website)
- any others?

#### Next week: 
- [OMI 100](https://github.com/omigroup/omigroup/discussions/371)!

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo
- `IPSME` and Me
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
New WG props

We may want to close down some of our WGs and create some new fresh ones. Some proposals include: 
- https://github.com/omigroup/omigroup/issues/368
- https://github.com/omigroup/omigroup/issues/372
- https://github.com/omigroup/omigroup/issues/209

**Comment by mrmetaverse** [comment]
## MSF updates 
- attended the oversight meeting for the Asia region participants 
- one slide asked how can we make sure we are inclusive in our membership tiers. 
- huge win: allowing groups like ours to participate in oversight without a membership fee. 
- Lets get it to policy: establishing a longlasting path for inclusivity like this. 
- took some time at the end to showcase [the gltf posters](https://github.com/madjin/gltf-extensions#posters) that aaron and jin and others made 
- Neil and other leaders were clearly impressed with OMI efforts in the space, and requested a "mind-meld" of OMI groups and MSF groups for more shared knowledge and collaborations. Talking through and showcasing what we are working on.

**Comment by madjin** [comment]
**MSF Delegates update**
I gave a presentation to the Interoperable Avatars/Characters Group on Monday, got the impression that people were blown away by my presentation since many stayed during the watercooler.


![VMagicMirror_wFiCgQtlWS](https://user-images.githubusercontent.com/32600939/225374150-cca45535-572d-4f85-b0eb-80083a646a07.png)

I showed off https://hyperfy.io/msf to avatar group yesterday too, describing how [infinite canvas tools](https://infinitecanvas.tools/) can help groups desilo their documentation, onboard faster, make meetings more engaging / productive even

![brave_xS53x7RLmD](https://user-images.githubusercontent.com/32600939/225374948-01c57e18-7ed3-4558-8d99-7169d918a1b5.png)
![brave_UvKrq4jFeS](https://user-images.githubusercontent.com/32600939/225374797-c47a91c4-ae47-4e49-a57d-e908676227a6.png)

I was impressed by the formality and organization of interoperable characters/avatars EG, but also have some feedback overall on how MSF is operating rn and where I think it should be going in terms of direction, will outline a constructive feedback blog post on some suggestions such as virtual expo / conference, mindmeld opportunities, etc.


![brave_ChEUouzh61](https://user-images.githubusercontent.com/32600939/225374841-5887ac03-1219-4869-91cf-ade534b43031.png)

**Comment by mrmetaverse** [comment]
"When it comes to world hops and wearables, I want us to be more about it instead of talk more about it." - Jin

how can people make standards for interoperable avatars if they don't have one themselves?

**Comment by mrmetaverse** [comment]
## Media 
- MTV looking for 1:1 time to meet to advance the media group 
- yesterday nobody was at the meeting so can we: 
  - ping people on discord when meetings start 
  - find a new time to meet as a group and remove the existing. 
-

**Comment by mrmetaverse** [comment]
## UX 
- want to meet? set up a meeting.

**Comment by mrmetaverse** [comment]
## OMI 100 planning
- https://github.com/omigroup/omigroup/discussions/371

**Reply by mrmetaverse** [reply]
check out the booth: https://hyperfy.io/neonstreet

**Comment by mrmetaverse** [comment]
## Meet Pixel in the OMI website 
Ant released the sickest update to our site yet! Pixel can answer questions about OMI, and how to get involved. 

1. visit https://omigroup.org/
2. scroll down to the footer 
3. click enter world 
4. click into the world and move around your mouse. 
5. walk up to the avatar "Pixel" and you can type a question. 
6. wait 1-3 seconds. 
7. Boom! Pixel answers to the best of their ability! 
![image](https://user-images.githubusercontent.com/63426722/225387312-e88a7d3c-b371-497f-811b-58cfd7fd4dbc.png)

**Comment by mrmetaverse** [comment]
Dancing with @codefrau 
![image](https://user-images.githubusercontent.com/63426722/225391149-8228a6f7-2273-48dc-aca3-4db6e5724d82.png)
